google/cloud/alloydb/connector/connector.py

```diff
@@ -149,13 +159,80 @@ async def connect_async(self, instance_uri: str, driver: str, **kwargs: Any) ->
 
         # synchronous drivers are blocking and run using executor
         try:
-            connect_partial = partial(connector, ip_address, context, **kwargs)
+            metadata_partial = partial(
```

Comment:
You'll also need to submit "useMetatadaExchange" in the certificate request above to get the proxy server to use the metadata exchange (instead of going straight to postgres).

Reply:
done